src/ontology/mondo-edit.obo

```diff
@@ -582755,6 +582755,18 @@ property_value: http://purl.org/dc/elements/1.1/date "2024-07-23T20:59:58Z" xsd:
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-0587-4693
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7831" xsd:anyURI
 
+[Term]
+id: MONDO:1040026
+name: metastatic malignant neoplasm in the brain
+def: "Any human metastatic malignant neoplasm affecting the brain." [PMID:29262201]
+is_a: MONDO:0001657 {source="PMID:29262201"} ! brain cancer
+is_a: MONDO:0024880 {source="PMID:29262201"} ! metastatic malignant neoplasm
+intersection_of: MONDO:0024880 ! metastatic malignant neoplasm
+intersection_of: disease_arises_from_feature UBERON:0000955 ! brain
```

Comment:
the relation should be "disease has location". 
Similar to 'metastatic malignant neoplasm in the eye' (MONDO:0044913) and 'metastatic malignant neoplasm in the spinal cord' (MONDO:0044912)

Reply:
Comment addressed